### PR TITLE
fix:[CORE-1788] Add parameter for policies with N days

### PR DESCRIPTION
**Pull request by ershad-paladin**
# Description

- added parameter for policies with N days, by adding policy params in table
- standardised the policyIds with N days parameter
- changed insert statements for cf_PolicyTable and cf_PolicyParams for the standardised(new) policy IDs
- deleted the records for old policyIds in cf_PolicyTable and cf_PolicyParams 

Fixes # (issue)
parameter for N days

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] execute the db script
- [ ] check policy details in admin policy for all policies with N days

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

File: installer/resources/pacbot_app/files/DB_Policy.sql [hunk 1]
```diff
@@ -3528,4 +3528,9 @@ UPDATE cf_PolicyTable SET severity='critical' WHERE policyId='Enable_Secure_Boot
 UPDATE cf_PolicyTable SET severity='high' WHERE policyId='Enable_VPC_Flow_Logs_and_IntraNode_Visibility';
 
 UPDATE cf_PolicyTable SET policyDisplayName = 'Set NSG flow log retention period >= N days' WHERE policyId ='Network_Security_Group_Flow_Log_retention_more_than_90_days';
-UPDATE cf_PolicyTable SET policyDisplayName='Restrict Inbound Traffic to RDP Port 3389' WHERE policyId='NaclPublicAccessPort_version-1_NACLWithPublicAccessForPort3389_networkacl';
\ No newline at end of file
+UPDATE cf_PolicyTable SET policyDisplayName='Restrict Inbound Traffic to RDP Port 3389' WHERE policyId='NaclPublicAccessPort_version-1_NACLWithPublicAccessForPort3389_networkacl';
+
+INSERT IGNORE INTO cf_PolicyParams (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('IamAccessKeyRotatedInEvery90Days_version-1_IamAccessKeyRotatedInEvery90Days_iamuser','rotationDuration','90','90','true','true','false','Key rotation duration','Specify duration in days for access key rotation');
```
Comment (plyubich) [inline installer/resources/pacbot_app/files/DB_Policy.sql:3533]: @ershad-paladin is it difficult to standardize the ids?

We have names that are synonyms for which we have:

In90days
GreaterThan90
More_than_ninety

If possible please come up with a common notation for the keys.

**Review by plyubich — CHANGES_REQUESTED** [review]

File: installer/resources/pacbot_app/files/DB_Policy.sql (hunk 1 above)
Comment (ershad-paladin) [reply installer/resources/pacbot_app/files/DB_Policy.sql:3533]: we have 6 policies for N days. For policyId change, I'd suggest to create a separate ticket and address all these policiesIds in that. PFB all the policyIds.

CheckInactiveIamUser_version-1_CheckInactiveIamUser_iamuser
Ec2StoppedInstanceForLong_version-1_Ec2StoppedInstanceForLong_ec2
IamAccessKeyRotatedInEvery90Days_version-1_IamAccessKeyRotatedInEvery90Days_iamuser
Network_Security_Group_Flow_Log_retention_more_than_90_days
Set_Retention_Days_Greater_Than_Ninety
kms_keys_rotation_period_within_90_days

Comment (plyubich) [reply installer/resources/pacbot_app/files/DB_Policy.sql:3533]: If the policy id change is not a breaking change let's do it in this ticket.

Comment (ershad-paladin) [reply installer/resources/pacbot_app/files/DB_Policy.sql:3533]: done

**Review by plyubich — APPROVED** [review]

**Review by santhosh-challa — APPROVED** [review]

**Pull request merged**